lockbox-ios/Store/DataStore.swift

```diff
@@ -8,22 +8,4 @@ class DataStore: BaseDataStore {
     public static let shared = DataStore()
 
     override func initialized() { }
-
-    override func unlock() {
-        func performUnlock() {
-            self.storageStateSubject.onNext(.Unlocked)
-            self.profile.reopen()
-            self.profile.syncManager?.beginTimedSyncs()
```

Comment:
Does the new code implement timed syncs?

 * Y -> \o/ 💯 
 * N -> file an issue?

Reply:
#902